src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,9 +4,73 @@
   pageTitle="Tabs"
 >
   <sky-docs-demo-page-summary>
-    Sample description.
+    Tabs are containers for dividing large amounts of content on a page.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
```

Comment:
Maybe a question for @Blackbaud-SteveBrush : Does the "reset" button support checkboxes? Maybe I'm missing something, but I'm not sure on how to reset my demo settings when using checkboxes. Might need to pair up with you on that.

Reply:
This is a bug. I fixed it in `3.0.0-alpha.29`. Let me know if it's still a problem!

Reply:
Yep, worked like a champ. Thanks!